### PR TITLE
Updated CRDs and CSV from the upstream Strimzi 0.29.0 release

### PR DESCRIPTION
**Pull request by ppatierno**
Updated CRDs and CSV from the upstream Strimzi 0.29.0 release.
Notice that the changes on the CRDs are more related to new jackson update upstream which serialized YAMLs differently, mostly for descriptions.

### CODE REVIEW

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml [hunk 1]
```diff
@@ -373,14 +373,16 @@ metadata:
...
     containerImage: registry.redhat.io/amq7/amq-streams-rhel8-operator@sha256:c1a98649ba33febede789ac26d541632ff979cc8817bdd9f6188ffa755e6c7ec
-    createdAt: 2022-02-12 23:02:00
+    createdAt: 2022-07-01 10:33:00
     description: Red Hat AMQ Streams is a massively scalable, distributed, and high performance data streaming platform based on Apache Kafka®.
     repository: https://github.com/strimzi/strimzi-kafka-operator
     support: Red Hat
     operators.openshift.io/infrastructure-features: |-
       ["Disconnected", "Proxy"]
     operators.openshift.io/valid-subscription: |-
       ["Red Hat Integration", "Red Hat AMQ"]
```
Comment (scholzj) [inline operator-metadata/manifests/bundle.clusterserviceversion.yaml:383]: I guess we should add `Red Hat Application Foundations` here?

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml [hunk 2]
```diff
@@ -373,14 +373,16 @@ metadata:
     categories: Streaming & Messaging
     certified: 'false'
     containerImage: registry.redhat.io/amq7/amq-streams-rhel8-operator@sha256:c1a98649ba33febede789ac26d541632ff979cc8817bdd9f6188ffa755e6c7ec
-    createdAt: 2022-02-12 23:02:00
+    createdAt: 2022-07-01 10:33:00
     description: Red Hat AMQ Streams is a massively scalable, distributed, and high performance data streaming platform based on Apache Kafka®.
     repository: https://github.com/strimzi/strimzi-kafka-operator
     support: Red Hat
     operators.openshift.io/infrastructure-features: |-
       ["Disconnected", "Proxy"]
```
Comment (scholzj) [inline operator-metadata/manifests/bundle.clusterserviceversion.yaml:381]: `proxy-aware` as discussed offline.

File: operator-metadata/manifests/strimziclusteroperator.configmap.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: amq-streams-cluster-operator
+  name: strimzi-cluster-operator
```
Comment (scholzj) [inline operator-metadata/manifests/strimziclusteroperator.configmap.yaml:4]: I think you should keepo the original name and change only the content. Or alternatively, you need to change the mount in the operator deployment:
```yaml
                - name: co-config-volume
                  configMap:
                    name: amq-streams-cluster-operator
```

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml (hunk 1 above)
Comment (ppatierno) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:383]: I was not aware of it, going to add.

File: operator-metadata/manifests/bundle.clusterserviceversion.yaml (hunk 2 above)
Comment (ppatierno) [reply operator-metadata/manifests/bundle.clusterserviceversion.yaml:381]: Yeah ... and I forgot!!

File: operator-metadata/manifests/strimziclusteroperator.configmap.yaml (hunk 3 above)
Comment (ppatierno) [reply operator-metadata/manifests/strimziclusteroperator.configmap.yaml:4]: right

**Review by scholzj — APPROVED** [review]

**Review by kyguy — APPROVED** [review]

**Pull request merged**